### DISCUSSION TITLE
Louça

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#335] [Imagens]
Post de referência; https://www.vish.blog.br/imagens/tirinhas/louca/

**Comment by brunoaugusto** [comment]
Deve ter dado trabalho escrever esse "NÃãÃÃoOo" todo torto

**Comment by one-eyed-willy** [comment]
Toda... vez... :pensive:

**Comment by maryon-telvanni** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/tirinhas/louca/#comment_1160591601>):

> Deve ter dado trabalho escrever esse "NÃãÃÃoOo" todo torto

Bom, eu não sou lá muito boa com Photoshop então, é, foi um pouquinho difícil, sim
